ShopperInsights/src/main/java/com/braintreepayments/api/findeligiblepayments/FindEligiblePaymentMethodDetails.kt

```diff
@@ -0,0 +1,29 @@
+package com.braintreepayments.api.findeligiblepayments
+
+import org.json.JSONObject
+
+/**
+ * Details of a specific payment method.
+ *
+ * @property canBeVaulted Indicates if the payment method can be saved for future transactions.
+ * @property eligibleInPayPalNetwork Indicates if the payment method is eligible in the PayPal network.
+ * @property recommended Indicates if this payment method is recommended for the shopper.
+ * @property recommendedPriority The priority ranking of this payment method if recommended.
+ */
+internal data class FindEligiblePaymentMethodDetails(
```

Comment:
More nitpick but can we drop the `Find` on all these classes? It doesn't align with other naming patterns (we don't typically start with a verb, most data/result objects are just nouns)